js/apps/admin-ui/src/clients/advanced/AdvancedSettings.tsx

```diff
@@ -160,6 +165,37 @@ export const AdvancedSettings = ({
               )}
             />
           </FormGroup>
+          {isDPoPEnabled && (
+            <FormGroup
+              label={t("oAuthDPoP")}
+              fieldId="oAuthDPoP"
+              hasNoPaddingTop
+              labelIcon={
+                <HelpItem
+                  helpText={t("clients-help:oAuthDPoP")}
+                  fieldLabelId="clients:oAuthDPoP"
+                />
+              }
+            >
+              <Controller
+                name={convertAttributeNameToForm<FormFields>(
+                  "attributes.dpop.bound.access.tokens"
```

Comment:
This looks like same issue like in your other PR (The `iss` PR). Maybe you can just insert `,` as suggested?

Reply:
Thank you I will fix it.